app.py

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/env python2.7
+
+"""
+Amity Room Allocation System
+Usage:
+  create_room <room_type> <room_name>
+  add_person <first_name> <second_name> <email> <FELLOW|STAFF> [wants_accommodation]
+  reallocate_person <email> <new_room_name>
+  load_people
```

Comment:
Does load_people load from a text file thus you need to add an argument to pick up the file name